server/src/tasks/relay.js

```diff
@@ -168,6 +172,8 @@ const relay = withWalletAccount(async (account, { walletAddress, methodName, met
         job.save()
       }
     } else {
+      job.attrs.data.status = 'failed'
```

Comment:
there should be blockNumber for failing as well I think

Reply:
you right, I fixed that.